presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -15,6 +15,18 @@ services:
     volumes:
       - ../../../../:/docker/volumes/presto
 
+  presto-master-ldap:
+    image: 'teradatalabs/centos6-java8-oracle-ldap'
+    hostname: presto-master
+    env_file:
+      - ../../../target/classes/presto.env
+    volumes:
+      - /etc/ldap/rootca
```

Comment:
This line means that you want to mount `/etc/ldap/rootca` file or directory from the host machine to your docker container. For example my Ubuntu doesn't have such directory 

```
[andrii-thinkpad:~]  ⌘ ls /etc/ldap/rootca
ls: cannot access '/etc/ldap/rootca': No such file or directory
```

I'm not sure why it is needed, and whether it will work on any host maching. Anyway it looks suspicious.